content/en/apps/guides/hosting/self-hosting.md

```diff
@@ -153,3 +153,16 @@ volumes:
 ```
 
 Be sure to check the available storage space regularly and expand your volume when needed
+
+## Backup
+
+As mentioned above, regular backups should be made of the `/srv` directory to have holistic and easy to restore copies of all important data and the current CHT version installed.  To backup just the data and not the CHT, make copies of `/srv/storage/medic-core/`.  This directory includes 4 key sub-directies:
```

Comment:
If you don't read this linearly (which I seldom do with any docs) then "as mentioned above" makes me think I've missed something. If where it's mentioned above is a pre-requisite to reading this section then link to it. If it's not, then just remove these three words...

```suggestion
Regular backups should be made of the `/srv` directory to have holistic and easy to restore copies of all important data and the current CHT version installed.  To backup just the data and not the CHT, make copies of `/srv/storage/medic-core/`.  This directory includes 4 key sub-directies:
```